var/spack/repos/builtin/packages/jdk/package.py

```diff
@@ -22,34 +22,35 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
-#------------------------------------------------------------------------------
+#
 # Author: Justin Too <too1@llnl.gov>
-#------------------------------------------------------------------------------
+#
 import distutils
-from distutils import dir_util
-from subprocess import call
 
 import spack
 from spack import *
 import llnl.util.tty as tty
 
+
 class Jdk(Package):
     """The Java Development Kit (JDK) released by Oracle Corporation
        in the form of a binary product aimed at Java developers."""
-    homepage = "http://www.oracle.com/technetwork/java/javase/downloads/index.html"
+    homepage = "http://www.oracle.com/technetwork/java/javase/downloads/index.html"  # noqa: E501
```

Comment:
Homepage should no longer need noqa now that #1259 has been merged. You don't have to remove it, but if you do, just rebase off of the current develop to get the latest run-flake8.